WORKSPACE

```diff
@@ -32,12 +32,12 @@ nixpkgs_package(
 
 cc_configure_custom(
     name = "local_config_cc",
-    gcc = "@gcc//:bin/gcc",
+    gcc = "@compiler//:bin/gcc",
```

Comment:
I don't know how well it works on Mac. Using cc rather than gcc should be portable to more systems.

Reply:
In this case `bin/gcc` is just a wrapper shell script that the Nix derivation creates.  Taking another look, the naming is probably more confusing (since it's actually clang on Mac) so I've renamed it to `bin/cc`.